package.json

```diff
@@ -48,6 +48,7 @@
     "eslint-plugin-jsx-a11y": "^6.7.1",
```

Comment:
![image](https://github.com/yusufobr/react-calculator/assets/93039359/c4f550de-f644-4e81-95b5-76a20b2234cd)

**[OPTIONAL]**

- [ ] It would be nice if you make your commit message capitalized to make your pull request more professional 👍